R/helper_main.R

```diff
@@ -134,22 +134,15 @@ read_column_synonyms <- function(synonym_file) {
 #' }
 export_data <- function(data, filename, output_root, suffix) {
     logDebug("Start export_data. Suffix = ", suffix, ".")
-    data %>%
-        write.csv(
-            file =
-                file.path(
-                    output_root,
-                    paste0(
-                        filename,
-                        suffix,
-                        ".csv"
-                    )
-                ),
-            row.names = F,
-            na = "",
-            fileEncoding = "UTF-16LE",
-            quote = T
-        )
+    data %>% arrow::write_parquet( sink =
```

Comment:
does arrow also omit the row.names? What about missing values? So compared to `write.csv` what about the parameters `row.names`, `na`?